### PR TITLE
added setup-opnsense

### PR DESCRIPTION
**Pull request by JiyanFontaine**
I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Jiyan Fontaine <jiyan.fontaine@outlook.de>

### CODE REVIEW

File: tutorials/setup-opnsense/01.en.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/setup-opnsense"
+slug: "ssetup-opnsense"
```
Comment (rook59) [inline tutorials/setup-opnsense/01.en.md:4]:
```suggestion
slug: "setup-opnsense"
```

File: tutorials/setup-opnsense/01.en.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+and does not want or want this and that. I can only recommend the official documentation from OPNsense.
+
+## Conclusion
+
+This article shows the necessary steps for setting up a OPNsense server with Hetzner Cloud.
+    
+    
+I have read and understood the Contributor's Certificate of Origin
+available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
+Signed-off-by: Jiyan Fontaine <jiyan.fontaine@outlook.de>
```
Comment (rook59) [inline tutorials/setup-opnsense/01.en.md:152]: Please remove this part and re-add the full commented block from the template

File: tutorials/setup-opnsense/01.en.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+
+1. Select new project and create a project name
+![Select Project Name](https://github.com/j1y4n/community-content/blob/master/tutorials/setup-opnsense/img/projectname.png)
+
+2. Select your project
+![Select Project](https://github.com/j1y4n/community-content/blob/master/tutorials/setup-opnsense/img/project.png)
+
+3. add server
+![Add Server](https://github.com/j1y4n/community-content/blob/master/tutorials/setup-opnsense/img/add_server.png)
```
Comment (rook59) [inline tutorials/setup-opnsense/01.en.md:47]: Please add a newline between enumeration and image, otherwise it might be on the same line which looks bad

**Review by rook59 — COMMENTED** [review]
Thank you for your submission.
Please note that this tutorial is currently very bare-bones, so while we can accept it, the credit will be minimal.

**Comment by j1y4n** [comment]
In the requirements it says that this has to be bare bone or have I misinterpreted that? How high would the credit be?

**Comment by rook59** [comment]
Can you please tell me where the requirements are that you mean here?

**Comment by j1y4n** [comment]
Hey, I would leave the tutorial as it is and take the lower credit.

**Comment by rook59** [comment]
@j1y4n please fix the outstanding issues, afterwards the tutorial can be marked as ready

**Comment by rook59** [comment]
@j1y4n There are still some outstanding issues. Please check again.

I also noticed the image links reference your fork `https://github.com/j1y4n/.../image.png`, please use `alt name` instead.

**Pull request closed without merging**